pymongo/encryption.py

```diff
@@ -64,7 +64,7 @@
 from pymongo.write_concern import WriteConcern
 
 _HTTPS_PORT = 443
-_KMS_CONNECT_TIMEOUT = 10  # TODO: CDRIVER-3262 will define this value.
+_KMS_CONNECT_TIMEOUT = 20  # TODO: CDRIVER-3262 will define this value.
```

Comment:
[CDRIVER-3262](https://jira.mongodb.org/browse/SPEC-1394) settled on this being the same as the default mongoclient connect timeout. Could you update the comment and use `common.CONNECT_TIMEOUT` here?